playwright.config.ts

```diff
@@ -49,6 +49,7 @@ const config: PlaywrightTestConfig = {
   webServer: {
     command: 'npm run preview',
     port: 3000,
+    reuseExistingServer: !process.env.CI,
```

Comment:
Is there any documentation we need to change to reflect this dev e2e vs built e2e change for test running?

Reply:
Yes! I'll update the docs. I completely forgot about that 🤦🏻

Reply:
Isn't the bit about the build in TESTING.md now outdated?
>All end-to-end tests assume a production build of the project is available if run from CI:
Since the `reuseExistingServer` will hook into the dev server unless in CI?